USAGE.md

```diff
@@ -167,3 +167,20 @@ For additional examples, check out existing manuscripts that use the Manubot (so
 + The Manubot 2018 Development Proposal ([source](https://github.com/greenelab/manufund-2018), [manuscript](https://greenelab.github.io/manufund-2018/))
 
 If you are using the Manubot, feel free to submit a pull request to add your manuscript to the list above.
+
+## Citing Manubot
+
+To cite the Manubot project or for more information on its design and history, see `@url:https://greenelab.github.io/meta-review/`:
```

Comment:
The `@url:https://greenelab.github.io/meta-review/` doesn't seem necessary, but it doesn't bother me if you want to keep it.

Reply:
I think it's useful for those who want to copy paste the reference.